src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -226,8 +238,14 @@ public static void WebSocket_Https_Duplex_BinaryStreamed()
                 "Test case timeout was reached while waiting for the stream response from the Service to be completed.");
             clientReceiver.ReceiveStreamCompleted.Reset();
 
+            // Getting results from server via callback.
+            client.GetLog();
+            Assert.True(clientReceiver.LogReceived.WaitOne(ScenarioTestHelpers.TestTimeout),
+            "Test case timeout was reached while waiting for the Logging from the Service to be received.");
+
```

Comment:
Nit: not needed to merge.  It would aid debugging to add to the message how long the TestTimeout was when the error occurred.  We might configure it differently in lab or debugging runs.